### DISCUSSION TITLE
Installation Unable (Windows 11 22h2)

### DISCUSSION THREAD

**Discussion by Eta-Beta** [Alex313031/Thorium-Win#117] [Q&A] [answered]
Hi everyone, I'm a Windows user as you can see, I downloaded the installer but it doesn't want to install itself, it starts but after a few seconds it seems like there's no one there anymore

Good morning, one update on my new laptop installed without problems, while the other one where it doesn't install is very old but I installed Windows 11 22H2, in any case it is an Asus P5QC Intel(R) Core(TM)2 processor Quad CPU Q6600 @ 2.40GHz, 2403 Mhz, 4 cores, 4 logical processors. Do you think this is the cause of the problem?

**Comment by gz83** [comment, marked as answer]
I think you need to install Thorium-SSE3 version on the old computer

https://github.com/Alex313031/Thorium-SSE3/releases/tag/M117.0.5938.157-1

**Reply by Eta-Beta** [reply]
Grazie mille ha funzionato 😍😍
